backend/services/table_anomaly_service.py

```
@@ -0,0 +1,433 @@
+            items = self.db_repository.container.query_items(
+                query=query,
+                parameters=parameters,
+                enable_cross_partition_query=True
+            )
+            
+            return list(items)
+        except Exception as e:
+            print(f"Error getting anomaly details: {e}")
+            return []
+
+    async def acknowledge_anomaly(self, detection_id: str, tenant_id: str = "default") -> Dict[str, Any]:
```

Ruff (0.13.1)

Unused method argument: `tenant_id`

(ARG002)